mythril/interfaces/cli.py

```diff
@@ -164,7 +164,7 @@ def main():
             address, _ = mythril.load_from_address(args.address)
         elif args.solidity_file:
             # Compile Solidity source file(s)
-            #if args.graph and len(args.solidity_file) > 1:
+            # if args.graph and len(args.solidity_file) > 1:
```

Comment:
Why is this commented out? Does generating call graphs from multiple files work?

Reply:
After a small glance at the code it doesn't seem that graph would work for multiple files, but this was commented out before this pr, i just added a space before '#' to make it pep-8 compliant :smiley: .We may probably need to uncomment this.

Reply:
Not sure who commented this out and why. Let's uncomment it?

Reply:
Removed